lib/3scale/backend/stats/partition_eraser_job.rb

```diff
@@ -0,0 +1,64 @@
+module ThreeScale
+  module Backend
+    module Stats
+      # TODO: from configuration
+      DELETE_BATCH_SIZE = 50
```

Comment:
In this PR, this constant is only used in `PartitionEraserJob`, so I think it should be defined there and declared private.
I haven't checked all the related PRs yet, so not sure if that will change.

Same for the `PARTITION_BATCH_SIZE` constant defined in the other job class.

Reply:
both `DELETE_BATCH_SIZE` and `PARTITION_BATCH_SIZE` are designed to be configuration parameters